2022/0xxx/CVE-2022-0419.json

```diff
@@ -1,88 +1,89 @@
 {
-    "CVE_data_meta": {
-        "ASSIGNER": "security@huntr.dev",
-        "ID": "CVE-2022-0419",
-        "STATE": "PUBLIC",
-        "TITLE": "NULL Pointer Dereference in radareorg/radare2"
-    },
-    "affects": {
-        "vendor": {
-            "vendor_data": [
-                {
-                    "product": {
-                        "product_data": [
-                            {
-                                "product_name": "n/a",
-                                "version": {
-                                    "version_data": [
-                                        {
-                                            "version_value": "n/a"
-                                        }
-                                    ]
-                                }
-                            }
-                        ]
-                    },
-                    "vendor_name": "n/a"
+  "CVE_data_meta": {
+    "ASSIGNER": "security@huntr.dev",
+    "ID": "CVE-2022-0419",
+    "STATE": "PUBLIC",
+    "TITLE": "NULL Pointer Dereference in radareorg/radare2"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "product": {
+            "product_data": [
+              {
+                "product_name": "radareorg/radare2",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_value": "6.0.0"
```

Comment:
Should be 5.6.0

Reply:
@trufae - please see https://github.com/CVEProject/cvelist/pull/4434